README.md

```diff
@@ -30,23 +30,32 @@ First, we need to get the GitHub API token for the bot account, which will be co
 export DANGER_GITHUB_API_TOKEN=XXXXX
 ```
 
-Second, if we use another Ruby tools and gems, we need to add the rules gem to our Gemfile and update the bundle.
+Second, if we do not use other Ruby tools and gems, we need to create Gemfile for our dependencies:
+
+```
+bundle init
+```
+
+Then we add the Danger rules gem as a dependency:
 
 ```
 bundle add thefuntasty_danger
 ```
 
-And then if we sometimes need to update our rules, we only need to call:
+And last if we need to install/update our rules, we only need to call:
 
 ```
-bundle update
+bundle [update]
```

Comment:
Why those brackets?

Reply:
I wanted to make clear that installing is made by calling just `bundle` and when you want to update dependencies it is made by calling `bundle update`. How do you think I should write this?

Reply:
`bundle [install|update]`. Optional character of `[update]` is much more obvious with this.